config.json

```diff
@@ -37,23 +37,51 @@
     "last 3 versions"
   ],
   "settings": {
-    "checkout_logo_position": "left",
-    "checkout_header_backgroundColor": "#fafafa",
-    "checkout_main_backgroundColor": "#ffffff",
-    "checkout_cart_backgroundColor": "#ffffff",
-    "checkout_headings_font": "Google_Montserrat_400",
-    "checkout_body_font": "Google_Karla_400",
-    "checkout_body_color": "#f010da",
-    "checkout_links_color": "#f010da",
-    "checkout_buttons_backgroundColor": "#454545",
-    "checkout_errors_color": "#f010da",
-    "checkout_formField_backgroundColor": "white",
```

Comment:
Please consider that these variables are already used by merchants. Renaming or changing a value could potentially break merchant themes